sdk/rust/src/processor/handler.rs

```diff
@@ -278,11 +278,11 @@ impl TransactionContext {
     /// * `addresses` - the addresses to fetch
     pub fn delete_state(
         &mut self,
-        addresses: Vec<String>,
+        addresses: &[String],
```

Comment:
Technically, this is a breaking API change. The Rust SDK is considered stable yet, so this is probably fine.